defaultentities.js

```diff
@@ -583,6 +642,12 @@ module.exports = { // ADD MORE (vanilla) AS YOU PLEASE (or modded if it's just f
     width: 2,
     height: 2
   },
+  flamethrower_turret: {
+    type: item,
+    width: 2,
+    height: 3
```

Comment:
Should this be 2?

Reply:
No. I checked in-game. 
The flamethrower has additional pipes behind it. It is 2x3. 
But perhaps it should be rotated.

Reply:
Nope, it is indeed correct, the wiki says it is 2x2 so I will fix that :)